examples/Advanced-Building-your-own-architecture.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>May be link to <code>explore-different-models</code> example notebook (https://github.com/NVIDIA-Merlin/models/blob/main/examples/Exploring-different-models.ipynb)  and mention `if you want to check the high-level implementation of DLRM, please refer to .. notebook'. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I added a link to the notebook and the DLRM architecture in the <code>Overview</code> section.  </p>